pkg/kubelet/cadvisor/util.go

```diff
@@ -42,6 +45,14 @@ func CapacityFromMachineInfo(info *cadvisorapi.MachineInfo) v1.ResourceList {
 			int64(info.MemoryCapacity),
 			resource.BinarySI),
 	}
+	if utilfeature.DefaultFeatureGate.Enabled(features.SupportNodePidsLimit) {
```

Comment:
won't this cause pids to show up in node capacity?

Reply:
I believe this is now fixed, if I understood you correctly.